pages/linux/playerctl.md

```diff
@@ -20,8 +20,12 @@
 
 - Send a command to a specific player:
 
-`playerctl --player={{player_name}} {{command}}`
+`playerctl --player = {{player_name}} {{command}}`
```

Comment:
Is it actually supposed to be this way? The previous one is the usual way command arguments are used.

Reply:
Spaces would cause it to be invalid, as far as I know. Perhaps we could remove this change,  @enes4949?